### ISSUE TITLE
Crash on selecting a series after a search

### ISSUE THREAD

**Issue by WanWizard** [kiddac/XStreamity#50]
On 4.59-20240915, OpenPLi Scarthgap.

````
Traceback (most recent call last):
  File "/usr/lib/enigma2/python/Components/ActionMap.py", line 57, in action
    res = self.actions[action]()
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/data/openpli/pli-oe/scarthgap/build/tmp/work/all-oe-linux/enigma2-plugin-extensions-xstreamity/git/image/usr/lib/enigma2/python/Plugins/Extensions/XStreamity/series.py", line 1528, in parentalCheck
  File "/data/openpli/pli-oe/scarthgap/build/tmp/work/all-oe-linux/enigma2-plugin-extensions-xstreamity/git/image/usr/lib/enigma2/python/Plugins/Extensions/XStreamity/series.py", line 1585, in next
  File "/data/openpli/pli-oe/scarthgap/build/tmp/work/all-oe-linux/enigma2-plugin-extensions-xstreamity/git/image/usr/lib/enigma2/python/Plugins/Extensions/XStreamity/series.py", line 256, in createSetup
  File "/data/openpli/pli-oe/scarthgap/build/tmp/work/all-oe-linux/enigma2-plugin-extensions-xstreamity/git/image/usr/lib/enigma2/python/Plugins/Extensions/XStreamity/series.py", line 426, in getSeasons
AttributeError: 'XStreamity_Categories' object has no attribute 'tmdb2'
[ePyObject] (CallObject(<bound method ActionMap.action of <Components.ActionMap.ActionMap object at 0xa7890210>>,('XStreamityActions', 'ok')) failed)
````

The error also happens if tmdb use is disabled / switched off in the global playlist settings. Also, some series still have a cover and information, again even if the use of tmdb is disabled.

**Comment by kiddac** [comment]
I don't seem to be able to replicate this error. 
Can you see if it still happens on v4.60

If tmdb is turned off, the image and info displayed is what is in your provider data. This is to be expected. 
We use tmdb, because its more up to date and allows multi-language response.
Provider data for vod and series is static.

**Comment by kiddac** [comment]
OK I know why now. Someone has shared a line with me that crashes. 
The API data on those crashing doesn't have a TMDB id in the series data from the API call. 
There is that many versions of xtream codes API doing the rounds now, that it is becoming very difficult as they dont all bring back the same data. 

I will try and amend my code to work around this difference

**Comment by kiddac** [comment]
this should now be fixed in v4.61
